src/views/GovernanceReviewTeam/GRBReview/DocumentsTable.tsx

```diff
@@ -0,0 +1,332 @@
+        accessor: ({ status }) => {
+          // Repurpose the accessor to use `status` for sorting order
+          if (status === SystemIntakeDocumentStatus.PENDING) return 1;
+          if (status === SystemIntakeDocumentStatus.AVAILABLE) return 2;
+          if (status === SystemIntakeDocumentStatus.UNAVAILABLE) return 3;
+          return 4;
+        },
+        Cell: ({ row }: CellProps<SystemIntakeDocument, string>) => {
+          // Show the upload status
+          // Virus scanning
+          if (row.original.status === SystemIntakeDocumentStatus.PENDING) {
+            return (
+              <em data-testurl={row.original.url}>
+                {t('technicalAssistance:documents.table.virusScan')}
+              </em>
+            );
+          }
+          // View or Remove
+          if (row.original.status === SystemIntakeDocumentStatus.AVAILABLE) {
+            // Show some file actions once it's available
+            const {
+              fileName: documentName,
+              canView,
+              canDelete,
+              url: documentUrl
+            } = row.original;
+
+            return (
+              <>
+                {
+                  /* View document */
+                  canView && (
+                    <Button
+                      type="button"
+                      unstyled
+                      onClick={() =>
+                        downloadFileFromURL(documentUrl, documentName)
+                      }
+                    >
+                      {t('technicalAssistance:documents.table.view')}
+                    </Button>
+                  )
+                }
+
+                {
+                  /* Delete document */
+                  canDelete && (
+                    <Button
+                      unstyled
+                      type="button"
+                      className="margin-left-2 text-error"
+                      onClick={() => setFileToDelete(row.original)}
+                    >
+                      {t('technicalAssistance:documents.table.remove')}
+                    </Button>
+                  )
+                }
+              </>
+            );
+          }
+          // Infected unavailable
+          if (row.original.status === SystemIntakeDocumentStatus.UNAVAILABLE) {
+            return t('technicalAssistance:documents.table.unavailable');
+          }
+          return '';
+        }
+      }
+    ];
+  }, [t]);
+
+  const table = useTable(
+    {
+      columns,
+      data: documents,
+      autoResetSortBy: false,
+      autoResetPage: true,
+      initialState: {
+        sortBy: useMemo(() => [{ id: 'uploadedAt', desc: true }], []),
+        pageIndex: 0,
+        pageSize: 5
+      }
+    },
+    useSortBy,
+    usePagination
+  );
+
+  const {
+    getTableBodyProps,
+    getTableProps,
+    headerGroups,
+    prepareRow,
+    page,
+    rows,
+    setPageSize
+  } = table;
+
+  const ConfirmDeleteModal = () => {
```

Comment:
This modal needs to be updated to match the [Figma](https://www.figma.com/design/G0UN35V7SY0g9nkTIESxsY/EASi-GRB-Experience?node-id=2603-59361&t=npq7sPo3VJgL8hKs-1).

Reply:
Woof, thanks I totally missed this.

Reply:
Added in [f2c84ba](https://github.com/CMS-Enterprise/easi-app/pull/2776/commits/f2c84ba292c27c814f60340ec34482be643dd977)